features/poller/monitors/core/blueprint-poller-monitors-core-minion.xml

```diff
@@ -0,0 +1,301 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
```

Comment:
I just added this file to make my build successful,  this file is already merged to main branch from HZN-656.